espnet/nets/pytorch_backend/transformer/positionwise_feed_forward.py

```diff
@@ -10,3 +10,17 @@ def __init__(self, d_model, d_ff, dropout):
 
     def forward(self, x):
         return self.w_2(self.dropout(torch.relu(self.w_1(x))))
+
+
+class PositionwiseConv1d(torch.nn.Module):
+    def __init__(self, d_model, d_ff, kernel_size, dropout):
```

Comment:
docstring

Reply:
Because this class does not depend on anything in this module, move this to a new individual module.

BTW, position-wise does not make sense here because conv1d is already position-wise (frame-wise). How about simply name `MultiLayerConv1d` or something?